frb_example/benches/dart/lib/env/web.dart

```diff
@@ -0,0 +1,3 @@
+// see https://github.com/flutter/flutter/issues/55870
+
+final sampleCount = int.fromEnvironment('SAMPLE_COUNT', defaultValue: 1000);
```

Comment:
nit: maybe we can merge the definition between web and io, by:

```
final sampleCount = int.parse(Platform.environment['SAMPLE_COUNT'] ?? 1000.toString()) ?? const int.fromEnvironment('SAMPLE_COUNT', defaultValue: 1000);
```

Reply:
that's what I did at first. but turns out `Platform.environment` and `T.fromEnvironment` does not exists both on each platform. Then, they yield different results based on:
- whether they are set with `export ENV=...` or `--define=ENV=...`
- whether the methods is used on dart vm or dart2js
I've already lost a couple of hours on it.

Reply:
Oh and did I mention that `environment` for example only exists on dart vm ? 😄 

<img width="1467" alt="Screen Shot 2565-10-07 at 21 29 30" src="https://user-images.githubusercontent.com/21016014/194578298-47fa8ad8-2279-4e34-b737-183b931ea307.png">

Reply:
I see. Weird behavior on dart...